pom.xml

```diff
@@ -79,6 +82,12 @@
             <artifactId>structs</artifactId>
             <version>1.5</version>
             <scope>test</scope>
+            <exclusions> <!-- coming from the core - RequireUpperBoundDeps fix -->
+                <exclusion>
+                    <groupId>org.jenkins-ci</groupId>
+                    <artifactId>annotation-indexer</artifactId>
```

Comment:
nit: I think you could update to structs 1.10+ to pick up https://github.com/jenkinsci/structs-plugin/commit/61baf803b832dce4fe5382ada874625a9a7bd6fc#diff-4773b53940bf034d91df6ea6b029028e which moves that dependency into provided scope instead, although that might necessitate other RequireUpperBoundDeps fixes.

Reply:
Yup, just update `structs`.